dev/workflow/generate_workflows.sh

```diff
@@ -328,9 +327,9 @@ EOM
       # Check if the specified yamls are for the specified system
       for i in "${!_nameref_yaml_list}"; do
          _yaml="${_nameref_yaml_list[${i}]}"
-         _found=$(grep -l "system: *${system}" "${_yaml_dir}/${_yaml}.yaml")
+         _found=$(grep -l "net: *${system}" "${_yaml_dir}/${_yaml}.yaml")
```

Comment:
It looks like I made a typo
```suggestion
         _found=$(grep -l "net: *${_net}" "${_yaml_dir}/${_yaml}.yaml")
```